javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -1273,6 +1273,27 @@ public static boolean isAnonymousConstructor(final MethodTree method) {
         return false;
     }
 
+    /**
+     * Determines if the tree is a call to Arrays.copyOf()
+     *
+     * @param tree tree to check
+     * @return true if the given tree is a call to Arrays.copyOf() method
+     */
+    public static boolean isArrayscopyOfMethodInvocation(MethodInvocationTree tree) {
```

Comment:
nit: in camel-case, this should be `isArraysCopyOfMethodInvocation` (that is, capitalize the "c" in "copy")